.github/workflows/run-tests.yml

```diff
@@ -20,23 +20,31 @@ jobs:
       - name: Show OS Info
         run: '[[ "$OSTYPE" == "linux-gnu"* ]] && { echo "OS Type: Linux"; (command -v lsb_release &> /dev/null && lsb_release -a) || cat /etc/os-release; uname -r; } || [[ "$OSTYPE" == "darwin"* ]] && { echo "OS Type: macOS"; sw_vers; uname -r; } || echo "Unsupported OS type: $OSTYPE"'
 
-      - name: Copy settings file
-        run: |
-          mkdir ~/.flowcept
-          cp resources/sample_settings.yaml ~/.flowcept
-          mv ~/.flowcept/sample_settings.yaml ~/.flowcept/settings.yaml
+#      - name: Copy settings file
+#        run: |
+#          mkdir ~/.flowcept
+#          cp resources/sample_settings.yaml ~/.flowcept
+#          mv ~/.flowcept/sample_settings.yaml ~/.flowcept/settings.yaml
```

Comment:
Just remove this from the file. Leaving it as commits just clutters up the file.